conans/assets/templates/new_v2_cmake.py

```diff
@@ -216,12 +216,12 @@ def get_cmake_lib_files(name, version, package_name="Pkg"):
                                                          package_name=package_name),
              "src/{}.cpp".format(name): source_cpp.format(name=name, version=version),
              "src/{}.h".format(name): source_h.format(name=name, version=version),
-             "src/CMakeLists.txt": cmake_v2.format(name=name, version=version),
+             "CMakeLists.txt": cmake_v2.format(name=name, version=version),
```

Comment:
Maybe we want to follow CMake common pattern that is the top root uses a ``add_subdirectory(src)`` and keep a CMakeLists.txt inside "src" with the heavy logic.

Reply:
Wouldn't having two `CMakeLists.txt` be too verbose? I mean, what would go into the root file, besides the `add_subdirectory()` call?

Reply:
It is verbose because it is a hello-world simple example. But it would probably be the most common layout out there in CMake projects. It will contain the project(), cmake_minimum_required() and the ``add_subdirectory()``

Reply:
Looks completely unnecessary to me, but if it's the consensus, who am I to disagree 😄 What do you think, @lasote ?

Reply:
I dont want to see two CMakeLists.txt in a hello world, please.